internal/resolve/catalog_test.go

```diff
@@ -673,7 +673,7 @@ func (w staticCatalogWalker) WalkCatalogs(ctx context.Context, _ string, f Catal
 			ObjectMeta: metav1.ObjectMeta{
 				Name: k,
 				Labels: map[string]string{
-					"olm.operatorframework.io/metadata.name": k,
+					"olm.operatorframework.io/name": k,
```

Comment:
Looks like a rebase mistake?

The correct label name is `olm.operatorframework.io/metadata.name` (we changed it recently)